fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```
@@ -1733,7 +1733,7 @@ struct RenameTenantImpl {
 			CODE_PROBE(true, "Metacluster rename old name not found");
 			throw tenant_not_found();
 		}
-
+		state bool canContinue = true;
```

It looks like this is only assigned in one place, in which case you could throw the error directly from there